docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -201,7 +202,7 @@ To query for a Patient's available `Documents`, you can either:
 ### Query for Documents using the API
 
 1. Start an asynchronous [Document Query](/medical-api/api-reference/document/start-document-query) across HIEs;
-1. Then, either process the inbound [Webhook message](/medical-api/more-info/webhooks) which will be sent if you
+2. Then, either process the inbound [Webhook message](/medical-api/more-info/webhooks) which will be sent if you
```

Comment:
nit: in markdown, we can make these all `1.` so that we don't have to worry about the numbering